src/pixie.c

```diff
@@ -27,8 +27,32 @@ void pixie_attack(void) {
 		dh_small ? "-S" : "-r" , dh_small ? "" : p->pkr);
 		printf("executing %s\n", cmd);
 		if (p->do_pixie < 0) {
-			pixie_popen(cmd);
-		} else {
+			int i;
+			char intervals[5][32] = {
+				" --start 00/0000 --end 00/0000",
```

Comment:
nope. i do not want this functionality inside reaver. it needs to be an external shell script.

Reply:
I get it.
Where should this external shell script be created? 
What is the most appropriate name for this external shell script?
How does reaver call this shell script?

Reply:
https://github.com/t6x/reaver-wps-fork-t6x/pull/273#issuecomment-469271484

i guess "pixiewps-wrapper" would be a good name, since it implies 1) it's a wrapper 2) calls pixiewps
it can be called with popen() as the semantics specified in my comment do not require usage of stdin or stderr.

Reply:
note that the wrapper could be implemented in any language (up to the user).
here's a very simple conforming version:
```sh
#!/bin/sh
if pixiewps "$@" > tmp.$$$ ; then
	cat tmp.$$$ | grep 'WPS pin:' | cut -d ':' -f 2 | sed 's/^[ ]\+//'
	rm tmp.$$$
	return 0
else
	rm tmp.$$$
	return 1
fi
```